addons/web/static/src/views/relational_model/record.js

```diff
@@ -727,7 +728,10 @@ export class Record extends DataPoint {
             const otherChanges = await this.model._onchange({
                 resModel: this.resModel,
                 resIds: this.resId ? [this.resId] : [],
-                changes: this._getChanges({ ...this._changes, ...changes }),
+                changes: {
+                    [this.config.relationField]: this._parent._getChanges(),
```

Comment:
how can this work? this._parent isn't always set

Reply:
Good question, I will check